### PR TITLE
Upgrade Google Could library bom to 16.3.0

### LINKED ISSUE #6696: Count BigQuery view more than once fails
Steps to reproduce:
1. Create a BigQuery view on BigQuery
2. Execute count(1) from view on Trino
3. Execute same query again and fail
 
Step 3's result of `BigQueryClient.query()` is different from the first call.

```bash
trino> select count(1) from bigquery.test.test_view;
 _col0
-------
     3

trino> select count(1) from bigquery.test.test_view;
Query 20210123_042935_00001_cbf4i failed: Internal error
java.util.NoSuchElementException
	at com.google.common.collect.AbstractIterator.next(AbstractIterator.java:153)
	at io.trino.plugin.bigquery.BigQuerySplitManager.createEmptyProjection(BigQuerySplitManager.java:139)
	at io.trino.plugin.bigquery.BigQuerySplitManager.getSplits(BigQuerySplitManager.java:93)
	at io.trino.split.SplitManager.getSplits(SplitManager.java:87)
	at io.trino.sql.planner.DistributedExecutionPlanner$Visitor.visitScanAndFilter(DistributedExecutionPlanner.java:203)
	at io.trino.sql.planner.DistributedExecutionPlanner$Visitor.visitTableScan(DistributedExecutionPlanner.java:185)
	at io.trino.sql.planner.DistributedExecutionPlanner$Visitor.visitTableScan(DistributedExecutionPlanner.java:156)
	at io.trino.sql.planner.plan.TableScanNode.accept(TableScanNode.java:143)
	at io.trino.sql.planner.DistributedExecutionPlanner$Visitor.visitAggregation(DistributedExecutionPlanner.java:303)
	at io.trino.sql.planner.DistributedExecutionPlanner$Visitor.visitAggregation(DistributedExecutionPlanner.java:156)
	at io.trino.sql.planner.plan.AggregationNode.accept(AggregationNode.java:203)
	at io.trino.sql.planner.DistributedExecutionPlanner.doPlan(DistributedExecutionPlanner.java:124)
	at io.trino.sql.planner.DistributedExecutionPlanner.doPlan(DistributedExecutionPlanner.java:131)
	at io.trino.sql.planner.DistributedExecutionPlanner.plan(DistributedExecutionPlanner.java:101)
	at io.trino.execution.SqlQueryExecution.planDistribution(SqlQueryExecution.java:470)
	at io.trino.execution.SqlQueryExecution.start(SqlQueryExecution.java:386)
	at io.trino.execution.SqlQueryManager.createQuery(SqlQueryManager.java:237)
	at io.trino.dispatcher.LocalDispatchQuery.lambda$startExecution$7(LocalDispatchQuery.java:143)
	at io.trino.$gen.Trino_testversion____20210123_042908_3.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)```

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #6696

Upgraded threetenbp to 1.5.0 and declared opencensus-api,commons-lang3and httpcore version explicitly because of upper bound.

Relevant issue: https://github.com/googleapis/java-bigquery/issues/802

### CODE REVIEW

**Review by wendigo — APPROVED** [review]
LGTM

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java [hunk 1]
```diff
@@ -133,6 +133,20 @@ public void testCountAggregationView()
                 "VALUES (1)");
     }
 
+    @Test(description = "regression test for https://github.com/trinodb/trino/issues/6696")
```
Comment (findepi) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java:136]: I'd
```suggestion
    /**
     * regression test for https://github.com/trinodb/trino/issues/6696
     */
    @Test
```

Comment (findepi) [reply plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java:136]: thanks for including a test!

File: plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java [hunk 2]
```diff
@@ -133,6 +133,20 @@ public void testCountAggregationView()
...
+    public void testRepeatCountAggregationView()
+    {
+        BigQuery client = createBigQueryClient();
+
+        String viewName = "test.repeat_count_aggregation_view";
```
Comment (findepi) [inline plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java:141]: use randomized name (randomTableSuffix). these tests are likely to be run on shared resources

**Review by findepi — APPROVED** [review]

**Pull request merged**